src/test/java/cloud/fogbow/ras/core/plugins/interoperability/openstack/network/v2/OpenStackNetworkPluginTest.java

```diff
@@ -76,240 +75,173 @@ public void setUp() throws InvalidParameterException, UnexpectedException {
+        OpenStackCloudUtils.getSGNameForPrivateNetwork(order.getInstanceId());
+        Mockito.verify(openStackNetworkPlugin, Mockito.times(TestUtils.RUN_ONCE)).retrieveSecurityGroupId(Mockito.any(), Mockito.any());
+        Mockito.verify(openStackNetworkPlugin, Mockito.times(TestUtils.RUN_ONCE)).doDeleteInstance(Mockito.any(), Mockito.any(), Mockito.any());
     }
 
-    //test case: Tests if an exception will be thrown in case that openstack raise an error when requesting for a new subnet.
+    //test case: test if the method makes the expected call.
     @Test
-    public void testRequestInstancePostSubnetError() throws IOException, FogbowException {
-        // set up
-        String createNetworkResponse = new CreateNetworkResponse(new CreateNetworkResponse.Network(NETWORK_ID)).toJson();
+    public void testDoGetInstance() throws FogbowException{
+        //setup
+        NetworkOrder order = createNetworkOrder(NETWORK_ID, TestUtils.DEFAULT_CIDR, DEFAULT_GATEWAY_INFO, NetworkAllocationMode.DYNAMIC);
+        Mockito.doReturn(order.getId()).when(openStackNetworkPlugin).doGetRequest(Mockito.any(), Mockito.any(), Mockito.any());
+        //exercise
+        openStackNetworkPlugin.doGetInstance(order, openStackV3User);
+        //verify
+        Mockito.verify(openStackNetworkPlugin, Mockito.times(TestUtils.RUN_ONCE)).doGetRequest(Mockito.any(), Mockito.any(), Mockito.any());
+    }
 
-        HttpResponse postNetworkResponse = new HttpResponse(createNetworkResponse, HttpStatus.SC_OK, null);
-        HttpResponse postSubnetResponse = new HttpResponse("", HttpStatus.SC_BAD_REQUEST, null);
-        Mockito.doReturn(postNetworkResponse).doReturn(postSubnetResponse).when(this.openStackHttpClient).
-                doGenericRequest(Mockito.any(HttpMethod.class), Mockito.anyString(), Mockito.any(HashMap.class),
-                        Mockito.any(HashMap.class), Mockito.any(OpenStackV3User.class));
+    //test case: test if the method make the expected call
+    @Test
+    public void testDoGetRequest() throws FogbowException, HttpResponseException{
+        //setup
+        Mockito.doReturn(TestUtils.EMPTY_STRING).when(openStackHttpClient).doGetRequest(Mockito.any(), Mockito.any());
+        //exercise
+        openStackNetworkPlugin.doGetRequest(openStackV3User, NETWORK_NEUTRONV2_URL_KEY, TestUtils.EMPTY_STRING);
+        //verify
+        Mockito.verify(openStackHttpClient, Mockito.times(TestUtils.RUN_ONCE)).doGetRequest(Mockito.any(), Mockito.any());
+    }
 
-//        Mockito.when(this.client.execute(Mockito.any(HttpUriRequest.class))).thenReturn(httpResponsePostNetwork,
-//                httpResponsePostSubnet, httpResponseRemoveNetwork);
-        NetworkOrder order = createEmptyOrder();
+    //test case: test if the method calls the ExceptionMapper when doGetRequest throws an Http exception.
+    @Test
+    public void testDoGetRequestInExceptionCase() throws FogbowException, HttpResponseException{
```

Comment:
We use "(un)successfully" suffix to the end of methods to mark it instead of "InExceptionCase". below methods should also follow this pattern.